charts/cloudzero-agent/README.md

```diff
@@ -79,124 +130,51 @@ prometheus-node-exporter:
+If running without the default exporters, adjust Prometheus scrape configs:
 
-custom-scrape-config.yaml
+`values-override.yaml modifying subchart values`
 ```yaml
 prometheusConfig:
   scrapeJobs:
-    kubeStateMetrics: # this disables the default kube-state-metrics scrape job, which will be replaced by an entry in additionalScrapeJobs
+    kubeStateMetrics:
       enabled: false
     additionalScrapeJobs:
     - job_name: custom-kube-state-metrics
-      honor_labels: true
-      honor_timestamps: true
-      scrape_interval: 1m
-      scrape_timeout: 10s
-      metrics_path: /metrics
       static_configs:
         - targets:
           - 'my-kube-state-metrics-service.default.svc.cluster.local:8080'
           - 'my-node-exporter.default.svc.cluster.local:9100'
       relabel_configs:
-      - separator: ;
-        regex: __meta_kubernetes_service_label_(.+)
-        replacement: $1
-        action: labelmap
-      - source_labels: [__meta_kubernetes_namespace]
-        separator: ;
-        regex: (.*)
-        target_label: namespace
-        replacement: $1
-        action: replace
       - source_labels: [__meta_kubernetes_service_name]
-        separator: ;
-        regex: (.*)
         target_label: service
-        replacement: $1
-        action: replace
-      - source_labels: [__meta_kubernetes_pod_node_name]
-        separator: ;
-        regex: (.*)
-        target_label: node
-        replacement: $1
-        action: replace
-      kubernetes_sd_configs:
-        - role: endpoints
-          kubeconfig_file: ""
-          follow_redirects: true
-          enable_http2: true
-
 ```
 
-This file can then be included with the helm install with the `-f custom-scrape-config.yaml` flag.
-
 ### Exporting Pod Labels
-Pod labels can be exported as metrics for use in the CloudZero platform by using the `metric-labels-allowlist` CLI argument to the `kube-state-metrics` container. This is disabled by default due to the increase in cardinality that exporting all pod labels introduces.
 
-To export **all** pod labels, set the following in the helm chart:
-```yaml
-kube-state-metrics:
-  extraArgs:
-    - --metric-labels-allowlist=pods=[*]
-```
-A subset of relevant pod labels can be included; as an example, exporting only the pod labels named `foo` and `bar` could be achieved with the following:
+Pod labels can be exported as metrics using `kube-state-metrics`. Customize the labels to export in `values-override.yaml`:
+
 ```yaml
 kube-state-metrics:
   extraArgs:
     - --metric-labels-allowlist=pods=[foo,bar]
 ```
-This is preferable to including all labels with `*` because the performance impact is lessened. Regular expression matching is not currently supported. See the `kube-state-metrics` [documentation](https://github.com/kubernetes/kube-state-metrics/blob/main/docs/developer/cli-arguments.md) for more details.
```

Comment:
I like removing this stuff, but I do think keeping this link would be good. users have to search for this KSM doc otherwise. this has a huge impact on resource usage, so might be good to keep a little more info

Reply:
done!